examples/native-fungible/index.html

```diff
@@ -226,43 +218,48 @@ <h2>Chain history for <code id="chain-id" class="hex">requesting a new microchai
               navigator.clipboard.writeText(document.querySelector('#account-id').textContent);
           });
 
+          console.warn(1);
+
           await linera.initialize();
           const faucet = await new linera.Faucet(import.meta.env.LINERA_FAUCET_URL);
           const mnemonic = ethers.Wallet.createRandom().mnemonic.phrase;
           const signer = linera.signer.PrivateKey.fromMnemonic(mnemonic);
           const wallet = await faucet.createWallet();
+          console.warn(5);
           const owner = signer.address();
           const chainId = await faucet.claimChain(wallet, owner);
-          const client = await new linera.Client(wallet, signer, false, true);
+          console.warn(6);
+          const client = await new linera.Client(wallet, signer);
+          console.warn(7);
           document.querySelector('#chain-id').innerText = chainId;
           document.querySelector('#owner').innerText = owner;
 
-          console.log('using application id:', import.meta.env.LINERA_APPLICATION_ID);
```

Comment:
Why are you removing this? I added it on purpose – very often we want to check what's the applicaton ID we're working with and it's nowhere to be found.

At the same time – you left lots of debugging console.warn-s